scripts/syncLegacyDB.js

```diff
@@ -47,36 +47,36 @@ db.settings(settings);
 //    PROBLEM:
 //    * cannot read un-approved ones but some can be inferred. (in the API but not in the map)
 async function getPhotosFromMap() {
-  const TIMERNAME= "reading from the map";
+  const TIMERNAME = "reading from the map";
   console.time(TIMERNAME);
   console.log("Getting photos from the map ...");
 
   const response = await request.get(config.map_url);
-  const lines = response.text.split('\n');
+  const lines = response.text.split("\n");
 
-  let searchPhase = 'latLng';
+  let searchPhase = "latLng";
   let photo = {};
   let photos = {};
   let numPhotos = 0;
 
   lines.forEach(line => {
     switch (searchPhase) {
-      case 'latLng':
+      case "latLng":
         if (line.match("var image_[0-9]")) {
-          const latitude = Number(line.match("\\[(-?[0-9]+\.[0-9]+),")[1]);
-          const longitude = Number(line.match(",(-?[0-9]+\.[0-9]+)\\],")[1]);
+          const latitude = Number(line.match("\\[(-?[0-9]+.[0-9]+),")[1]);
+          const longitude = Number(line.match(",(-?[0-9]+.[0-9]+)\\],")[1]);
           const location = new admin.firestore.GeoPoint(latitude, longitude);
 
           photo.location = location;
-          searchPhase = 'url'
+          searchPhase = "url";
         }
         break;
 
-      case 'url':
-        if (line.match("popup\.setContent")) {
```

Comment:
@munroTom  Are you sure it won't break the Regex ?

Reply:
@sebastianovide I could leave it out of this file for now as it seems to be a script that's run separately?
 I had a click through the app after I did it and then did some more work over the weekend and everything seemed to work as expected. I just found it easier cause I like using prettier and it means that I can keep track of the changes made a bit easier in the diffs if prettier is run over everything in one go